pkg/appgw/test_fixtures.go

```diff
@@ -139,236 +134,6 @@ func newCertsFixture() map[string]interface{} {
 	return toAdd
 }
 
-func newIngressBackendFixture(serviceName string, port int32) *v1beta1.IngressBackend {
```

Comment:
@rheaj99 I see that we moved the constants defined above this line to pkg/tests
This is great!
Can we also delete the `const` block above now?
If some of the constants above are still needed in the `appgw` package - we can use them from `tests`
We'd have to capitalize these consts so they are exorted and then `tests.NodeName` for instance.  And because the package name already explains what these constants are - we can drop the `testFixtures...` prefix